test/unit/dataLoaderUtil.test.js

```diff
@@ -0,0 +1,864 @@
+/* eslint-env mocha */
+import axios from 'axios';
+import MockAdapter from 'axios-mock-adapter';
+import sinon from 'sinon';
+import { expect } from 'chai';
+import dataLoaderUtil from '@dataLoaderUtil';
+import Actions from '@Actions';
+import Bib from '@Bib';
+import Search from '../../src/server/ApiRoutes/Search';
+import Hold from '@Hold';
+import routeMethods from '../../src/server/ApiRoutes/RouteMethods';
+
+describe('dataLoaderUtil', () => {
+  describe('server side call', () => {
+    describe('non-matching path', () => {
+      let axiosSpy;
+      let actionsSpy;
+      let sandbox;
+      let mockRouteMethods;
+      before(() => {
+        sandbox = sinon.createSandbox();
+        axiosSpy = sandbox.spy(axios, 'get');
+        mockRouteMethods = {
+          bib: sandbox.spy(),
+          search: sandbox.spy(),
+          hold: sandbox.spy(),
+        };
+        actionsSpy = [];
+        Object.keys(Actions).forEach((key) => {
```

Comment:
You can also try the `for ... in` loop instead of doing `Object.keys`. And also getting `(key, value)` and then doing `if (typeof value === 'function') {`.